README.md

```diff
@@ -153,6 +154,27 @@ There should now be a `application-secrets.properties` file in the `packages/ser
 ```
 </details>
 
+### Running only only for the Data Services
+
+A self-contained "lean" version docker-compose-lean.yml with all services necessary to run the `terarium` backend can be spun up with the following:
+
+```shell
+docker compose --file containers/docker-compose-lean.yml pull
+docker compose --file containers/docker-compose-lean.yml up --detach --wait
+```
+
+This will deploy a local `terarium` backend along with all required services along with a localized `keycloak` instance to auth against. In order to hit the data service endpoints on port 3000 you will need to include the following basic auth credential:
+
+```
+'Authorization: Basic YXBpLXNlcnZpY2U6aWNvbiB2aW9sZXQgcmh5bWU='
```

Comment:
we should have a fake credentials, or remove this.

Reply:
or at least change the service account in keycloak with a different username and password for the local keycloak

Reply:
It is, thats for the local one only